Library/Formula/terraform.rb

```diff
@@ -23,72 +32,104 @@ class Terraform < Formula
     url "https://go.googlesource.com/tools.git", :revision => "2a105dc4ba99792ac380f48d154af4e10b822846"
   end
 
-  go_resource "github.com/mitchellh/iochan" do
-    url "https://github.com/mitchellh/iochan.git", :revision => "b584a329b193e206025682ae6c10cdbe03b0cd77"
+  go_resource "code.google.com/p/go-uuid" do
+    url "https://code.google.com/p/go-uuid", :revision => "35bc42037350f0078e3c974c6ea690f1926603ab", :using => :hg
   end
 
-  go_resource "github.com/hashicorp/hcl" do
-    url "https://github.com/hashicorp/hcl.git", :revision => "513e04c400ee2e81e97f5e011c08fb42c6f69b84"
+  go_resource "google.golang.org/api" do
+    url "https://code.googlesource.com/api.git", :revision => "2f6114897375589857c508d7392e55d5e7580db8"
```

Comment:
Looks like this no longer exists: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24015/version=yosemite/testReport/junit/brew-test-bot/yosemite/fetch_terraform/